superset-frontend/src/dashboard/components/SliceHeaderControls/SliceHeaderControls.test.tsx

```diff
@@ -0,0 +1,191 @@
+// beforeEach(()=>{
+//   jest.resetAllMocks();
+// })
+
+test('Should render', () => {
+  const props = createProps();
+  render(<SliceHeaderControls {...props} />, { useRedux: true });
+  expect(screen.getByRole('button', { name: '...' })).toBeInTheDocument();
+  expect(screen.getByTestId('NoAnimationDropdown')).toBeInTheDocument();
+});
+
+test('Should render default props', () => {
+  const props = createProps();
+
+  // @ts-ignore
+  delete props.forceRefresh;
+  // @ts-ignore
+  delete props.toggleExpandSlice;
+  // @ts-ignore
+  delete props.exploreChart;
+  // @ts-ignore
+  delete props.exportCSV;
+  // @ts-ignore
+  delete props.cachedDttm;
+  // @ts-ignore
+  delete props.updatedDttm;
+  // @ts-ignore
+  delete props.isCached;
+  // @ts-ignore
+  delete props.isExpanded;
+  // @ts-ignore
+  delete props.sliceCanEdit;
+
+  render(<SliceHeaderControls {...props} />, { useRedux: true });
+
+  userEvent.click(screen.getByRole('menuitem', { name: 'Maximize chart' }));
+  userEvent.click(screen.getByTestId('refresh-chart-menu-item'));
+  userEvent.click(
+    screen.getByRole('menuitem', { name: 'Toggle chart description' }),
+  );
+  userEvent.click(
+    screen.getByRole('menuitem', { name: 'View chart in Explore' }),
+  );
+  userEvent.click(screen.getByRole('menuitem', { name: 'Export CSV' }));
+  userEvent.click(screen.getByTestId('refresh-chart-menu-item'));
+
+  expect(screen.getByRole('button', { name: '...' })).toBeInTheDocument();
+  expect(screen.getByTestId('NoAnimationDropdown')).toBeInTheDocument();
+});
+
+test('Should "export to CSV"', () => {
+  const props = createProps();
+  render(<SliceHeaderControls {...props} />, { useRedux: true });
+
+  expect(props.exportCSV).toBeCalledTimes(0);
+  userEvent.click(screen.getByRole('menuitem', { name: 'Export CSV' }));
+  expect(props.exportCSV).toBeCalledTimes(1);
+  expect(props.exportCSV).toBeCalledWith(371);
+});
+
+test('Should "View chart in Explore"', () => {
+  const props = createProps();
+  render(<SliceHeaderControls {...props} />, { useRedux: true });
+
+  expect(props.exploreChart).toBeCalledTimes(0);
+  userEvent.click(
+    screen.getByRole('menuitem', { name: 'View chart in Explore' }),
+  );
+  expect(props.exploreChart).toBeCalledTimes(1);
+  expect(props.exploreChart).toBeCalledWith(371);
+});
+
+test('Should "Toggle chart description"', () => {
+  const props = createProps();
+  render(<SliceHeaderControls {...props} />, { useRedux: true });
+
+  expect(props.toggleExpandSlice).toBeCalledTimes(0);
+  userEvent.click(
+    screen.getByRole('menuitem', { name: 'Toggle chart description' }),
+  );
+  expect(props.toggleExpandSlice).toBeCalledTimes(1);
+  expect(props.toggleExpandSlice).toBeCalledWith(371);
+});
+
+test('Should "Force refresh"', () => {
+  const props = createProps();
+  render(<SliceHeaderControls {...props} />, { useRedux: true });
+
+  expect(props.forceRefresh).toBeCalledTimes(0);
+  userEvent.click(screen.getByTestId('refresh-chart-menu-item'));
```

Comment:
```suggestion
  userEvent.click(screen.getByRole('menuitem', { name: /Force refresh/ }));
```

Reply:
You can also remove `data-test="refresh-chart-menu-item"` in `index.jsx`

Reply:
Since I didn't add it, I prefer not to remove it.